src/common/lock.c

```diff
@@ -280,3 +287,58 @@ lockRelease(bool failOnNoLock)
 
     FUNCTION_LOG_RETURN(BOOL, result);
 }
+
+/**********************************************************************************************************************************/
+bool
+writeLockPercentageComplete(const String *lockPath, const String *stanza, const String *execId, const double percentageComplete)
```

Comment:
All extern'd functions in this module should begin with "lock". So `lockWritePercentageComplete`. Also I think percentage is more verbose than needed. How about using "percent" (here and in variable names)?

Reply:
All pointer params should be const.

Reply:
updated in 0f56ed01..1f3fbd2b